scripts/generate-rpm-specs.sh

```diff
@@ -4,8 +4,8 @@
 # spec => rpm files: run rpmbuild -bb *.spec
 # install rpm: yum localinstall /root/rpmbuild/RPMS/x86_64/*.rpm
 
-for n in {1..100}; do
-    cat >pkg-$(printf "%03d" ${n}).spec <<EOF
+for ((n=1; n<=$1; n++)); do
```

Comment:
I suggest we fail if `$1` is not specified.

```suggestion
pkgcount=${1:?missing package count}
for n in $(seq 1 $pkgcount); do
```